src/model/todoList.js

```diff
@@ -1,11 +1,11 @@
-
+const Utils = require('../utils');
 class TodoList {
     constructor(id, title, creator) {
         this.id = id
         this.title = title
         this.creator = creator
         this.list = []
-        this.hashVersion = null // TODO: CREATE HASH
+        this.hashVersion = Utils.generateRandomHash()
```

Comment:
@vaalecav Sabes que me quede pensando, este generateHash en el constructor no pisaria el hash en algun momento que estemos instanciando algo por mas que no se haya modificado la lista? Ponele en un createList de un commit o algo asi

Reply:
@vaalecav Recien probando lo del retry me fallo esto del hash cuando tenes multiples nodos.

**Steps to reproduce**:
- Levantate el registry
- Metele varios nodos, tipo 4 o 5
- Crea una lista (esto anda ok)
- **Trata de editar la lista un par de veces:** Aca es donde falla, si te fijas los hash que genera, son distintos en cada commit (o sea en cada nodo) y el frontend puede impactar medio random contra cualquier nodo, si no le pega al mismo nodo, el hash que tiene el front es distinto y falla

Reply:
Pero en un `createList` tiene sentido que se genere el hash de la lista, estas creando una lista por ende se le crea un hash asociado.

Reply:
Sobre el segundo comment, no entiendo cual es el error, no es eso lo que buscamos? Que en caso de que tenga una versión desactualizada solicitarle que recargue la página.